### ISSUE TITLE
Use RegExp in json expect type

### ISSUE THREAD

**Issue by renanpallin** [vlucas/frisby#480]
Wouldn't be nice if we could do:

```
 frisby.fetch(testHost + '/users/1')
      .expect('json', 'email', /joe\.\w+@\w+\.\w{3}/)
      .done(doneFn);
```
The only way today we can use regex is with bodyContains, but sometimes a validation in field level would be better in my opinion.
A problem that I've faced is when someone that does not know that a regex can't be used here uses, and the test pass, like:

 frisby.fetch(testHost + '/users/1')
      .expect('json', 'email', /another_thing{3}/) // not even a valid e-mail
      .done(doneFn);

This should not pass, but passes with the way that json expect type works today.

**Comment by renanpallin** [comment]
Don't know If I should wait for an release in npm with this improvement to close the issue =P

**Comment by vlucas** [comment]
This has been released in the most recent version `v2.1.0`.
